### PR TITLE
Drop dependencies for SP1 and initial support for SP2

### PR DESCRIPTION
**Pull request by aplanas**

### CODE REVIEW

**Comment by jsuchome** [comment]
Looks correct, not sure if anything else is not missing, but I don't have idea about anything.

+1

File: chef/data_bags/crowbar/template-provisioner.json [hunk 1]
```diff
@@ -172,7 +189,7 @@
     "provisioner": {
       "crowbar-revision": 0,
       "crowbar-applied": false,
-      "schema-revision": 35,
+      "schema-revision": 36,
```
Comment (rsalevsky) [inline chef/data_bags/crowbar/template-provisioner.json]: We should skip some revisions here to have some free space for Cloud 6.

**Comment by aplanas** [comment]
* Jump migration to 50

**Comment by jsuchome** [comment]
+1 after change of migration number

**Comment by toabctl** [comment]
+1

**Comment by rsalevsky** [comment]
+1

**Comment by aplanas** [comment]
gating fails because manila (unrelated)

**Pull request merged**